testing/scenario_app/ios/Scenarios/ScenariosUITests/iPadGestureTests.m

```diff
@@ -196,6 +199,101 @@ - (void)testPointerHover {
   XCTAssertTrue([app.textFields[removeMessage] waitForExistenceWithTimeout:1],
                 @"PointerChange.remove event did not occur for a hover");
 }
+
+- (void)testPointerScroll {
+  BOOL supportsPointerInteraction = NO;
+  SEL supportsPointerInteractionSelector = @selector(supportsPointerInteraction);
+  if ([XCUIDevice.sharedDevice respondsToSelector:supportsPointerInteractionSelector]) {
+    supportsPointerInteraction =
+        performBoolSelector(XCUIDevice.sharedDevice, supportsPointerInteractionSelector);
+  }
+  XCTSkipUnless(supportsPointerInteraction, "Device does not support pointer interaction.");
```

Comment:
do you know if any of our CI test devices supports pointer interaction?

Reply:
This testing support depends on iOS 15 anyway, so I don't think it's run yet. Right now I believe the simulator in CI is iPhone 8 on iOS 13. We need iPad on iOS15+.

Reply:
@jmagman probably knows who can help with setting up test devices for iOS15

Reply:
We can do this in the simulator, we shouldn't need physical devices, here:
https://github.com/flutter/engine/blob/92ad24a2ae3bf6cd8a0766b32872e7771ea91447/testing/scenario_app/run_ios_tests.sh#L43

We need to get https://github.com/flutter/flutter/issues/85558 done so these tests actually run 🙂 
It's fine to just match the pattern of the tests around it for now, as long as it passes for you locally.